src/spl/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 set(SOURCES
         square_generator.cc
         surface_generator.cc
-        b_spline.h)
+        spline.h)
```

Comment:
It's because it makes it easier to work in Clion (there are automatic checks running over the code when the file is included in the Cmake file) and I just forgot to remove it afterwards.